build/Dockerfile

```diff
@@ -25,6 +25,7 @@ ENV PACKAGES="\
     ruby-rdoc \
     ruby-irb \
     git \
+    openssh-client
```

Comment:
```suggestion
    openssh-client \
```

Reply:
Hi @Lowess can you please apply the change suggested by @webknjaz ?